### PR TITLE
Update scripts.js

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
There seems to be a trailing space. Reformat the HP IV line. Everything besides that seems good.

**Comment by DeathlyPlays** [comment]
Yeah travis build

File: mods/essb/scripts.js [hunk 1]
```diff
@@ -84,6 +84,14 @@ exports.BattleScripts = {
 					spa: 252,
 					spd: 208,
 				},
+				ivs: {
+					hp: 31,
```
Comment (DeathlyPlays) [inline mods/essb/scripts.js:88]: Remove that extra space after the hp: 31,

Comment (ghost) [reply mods/essb/scripts.js:88]: Ill take care of that

Comment (DeathlyPlays) [reply mods/essb/scripts.js:88]: Ah mk

**Pull request merged**